### DISCUSSION TITLE
Typescript absolute import from `src`

### DISCUSSION THREAD

**Discussion by tnorlund** [gatsbyjs/gatsby#30742] [Help]
I'm typescript-ifying my blog. I've enabled absolute imports from `src` in `Gatsby-node.js`:

```js
// Enable absolute imports from `src`.
// See https://gatsbyjs.org/docs/add-custom-webpack-config#absolute-imports.
exports.onCreateWebpackConfig = ( { actions } ) => {
  actions.setWebpackConfig( {
    resolve: {
      modules: [ path.resolve( __dirname, `src` ), `node_modules` ],
    },
  } )
}
```

VSCode is warning me that it can't find the type declarations, but it runs fine. Is there something I can do to tell VSCode where to look for the type declarations?

**Comment by lukethacoder** [comment]
ran into a similar issue. realised I needed a `tsconfig.json` file.

adding `"baseUrl"` to the `tsconfig.json` file seems to keep the VS Code warnings silenced. 

```json
{
  "compilerOptions": {
    "baseUrl": "./src/"
  },
}
```
